addons/web/static/tests/views/fields/priority_field_tests.js

```diff
@@ -639,4 +639,33 @@ QUnit.module("Fields", (hooks) => {
         await click(stars[stars.length - 1]);
         assert.verifySteps(["write"]);
     });
+
+    QUnit.test("PriorityField - prevent auto save with auto_save option", async function (assert) {
+        await makeView({
+            type: "form",
+            resModel: "partner",
+            resId: 1,
+            serverData,
+            arch: `
+                <form>
+                    <sheet>
+                        <group>
+                            <field name="selection" widget="priority" options="{'auto_save': False}"/>
+                        </group>
+                    </sheet>
+                </form>`,
+            mockRPC(_route, { method }) {
+                if (method === "write") {
+                    throw new Error("write should not be called");
```

Comment:
by convention, we now prefer using `assert.step("write");` than throwing an error.